parm/config/gefs/config.extractvars

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env bash                                                                                                                                                                                                                     
+
+########## config.extractvars ##########
+# Extractvars specific
+
+echo "BEGIN: config.extractvars"
+
+. "${EXPDIR}/config.resources" extractvars
+
+export compress_ocn=0 #1: compress extracted ocean product, 0: do not compress extracted ocean product
+export compress_ice=0 #1: compress extracted ice product, 0: do not compress extracted ice product
```

Comment:
Is compression only warranted for ocean and ice components (and not ATM)?

Reply:
Compression is only an option for the ocean and ice components because compression has not been highly needed for the atmosphere component. Since the atmosphere products have been in grib2 format, further compression has not been a priority because grib2 is already a very compressed format. Since there has been an interest for the ocean and ice products to be in netcdf format, compression has been more of a necessity for these components because netcdf is not as compressed of a data format. If there is an interest, we can add bzip2 compression as an option for the atmosphere component.